### PR TITLE
A first atempt at lower and upper bound, there is for sure some logic…

### PR DESCRIPTION
**Pull request by OskarSundberg**
…al problem in this. It's to late for this kind of work tbh.

### CODE REVIEW

File: report/lowerbound.tex [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Assumption I: The distance between any floor N and N+1 or N and N-1 is one and the time required is one. 
+
+Assumption II: Following I, a person need to go from floor X to floor Y, the Dn is equal to |X-Y|.
+
+Assumption III: The elevator has the capacity of N. 
+
+Assumption IV: For N persons that starts on the same floor and needs to go to the same floor, the lowest time spent waiting is equal to Dn*N.
+
+Prof:
```
Comment (Linkan42) [inline report/lowerbound.tex:13]: Proof

File: report/lowerbound.tex [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+A more general case: 
+A building with 2 floor and N person that need to move from floor zero to floor one. The optimal solution is, N*Dn = N*Dt = Time = N.
+
+This is the lower bound of our problem, by proving IV, it can be shown that the lower bound is Dn*N. 
+
+The upper bound:
+To prove the upper bound the following solution can be imagined.   
+
+A building whit F = 3, N = 1, the person is on floor 0 and like to go to floor 2. The elevator picks up the person but never goes to floor 2, instead the elevator moves between floor zero and floor one for ever. Resulting is  1 * \infty  =   \infty.
```
Comment (Linkan42) [inline report/lowerbound.tex:27]: with

File: report/lowerbound.tex [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Diffing a lower bound:
+
+Dn = distance needed, Dt = distance travled, N is the number on persons in the building and F = number off floors.(using zero index)
+
+Assumption I: The distance between any floor N and N+1 or N and N-1 is one and the time required is one. 
+
+Assumption II: Following I, a person need to go from floor X to floor Y, the Dn is equal to |X-Y|.
+
+Assumption III: The elevator has the capacity of N.
```
Comment (Linkan42) [inline report/lowerbound.tex:9]: You define N as number of persons in building, floor and elevator capacity ??

File: report/lowerbound.tex [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Assumption III: The elevator has the capacity of N. 
+
+Assumption IV: For N persons that starts on the same floor and needs to go to the same floor, the lowest time spent waiting is equal to Dn*N.
+
+Prof: 
+To prove Assumption IV: 
+Trivial case: A building with F = X and N = 0, there is no need to move the elevator. Resulting in 0*N=0, the Dn*0 = time = 0.
+
+Simple case: A building with F = 2 and N = 1, were the person needs to move for floor zero to floor one. It is trivial to see that the best solution is for the elevator to take one person from floor 0 to floor one. Resulting in Dn = time = 1 = Dt*1.
```
Comment (Linkan42) [inline report/lowerbound.tex:17]: where

**Review by Linkan42 — CHANGES_REQUESTED** [review]

**Comment by JoelScarinius** [comment]
Need to run git pull origin main

**Review by JoelScarinius — APPROVED** [review]
LGTM!

**Review by Linkan42 — APPROVED** [review]

**Pull request merged**